### ISSUE TITLE
Racket 5 tests don't work

### ISSUE THREAD

**Issue by clacke** [stchang/graph#39]
I'm aware that this is a known issue, as they are marked allowed to fail on Travis, but perhaps they should be removed or fixed? I'm making an attempt.

**Comment by clacke** [comment]
First, I replaced `raco pkg install --deps search-auto -n graph` with `raco pkg install --deps search-auto ../graph`. This made Racket 5 try to install the package. Then I ran into:

        ssl-connect: connect failed (error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure)

It is my quick guess that the Rackets installed by greghendershott/travis-racket are compiled with an SSL that is no longer compatible with the package servers.

It's picking the installers from https://mirror.racket-lang.org/installers , so I guess that should be considered the root cause. I accept that they are historical files and maybe won't be updated, so maybe this is an issue on greghendershott/travis-racket to use newer compiles of the old Rackets.
